web/packages/teleport/src/Main/MainContainer.tsx

```diff
@@ -29,11 +29,9 @@ export const MainContainer = styled.div`
   --sidebar-width: 256px;
   --sidenav-width: 76px;
   --sidenav-panel-width: 224px;
+  overflow: hidden;
   margin-top: ${p => p.theme.topBarHeight[0]}px;
   @media screen and (min-width: ${p => p.theme.breakpoints.small}px) {
     margin-top: ${p => p.theme.topBarHeight[1]}px;
   }
-  @media screen and (min-width: ${p => p.theme.breakpoints.large}px) {
-    margin-top: ${p => p.theme.topBarHeight[2]}px;
-  }
```

Comment:
Why don't we need this margin anymore at this size?

Reply:
because we've removed the 72px topbar size, which is what this margin was accommodating for